src/main/java/com/std/tothebook/api/entity/Book.java

```diff
@@ -0,0 +1,83 @@
+package com.std.tothebook.api.entity;
+
+import lombok.Builder;
+import lombok.Getter;
+
+import javax.persistence.Entity;
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+@Entity
+@Getter
+@Table(name="book")
+public class Book {
+
+    //id
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
```

Comment:
```suggestion
    private long id;
```
id가 테이블에는 bigint로 설정되어 long이 맞을 듯 합니다.